packages/ag-charts-community/src/chart/themes/constants.ts

```diff
@@ -7,14 +7,16 @@ import type {
     FontWeight,
 } from '../../options/agChartOptions';
 
-export const FONT_SIZE = {
+export const FONT_SIZE: Record<string, number> = {
```

Comment:
This removes the type safety so if you do `FONT_SIZE.NotASize`, it won't complain